persistence-api/src/main/java/io/stargate/db/query/builder/BuiltDML.java

```diff
@@ -278,7 +268,21 @@ private void handleTimestamp() {
     private void handleConditions() {
       for (BuiltCondition bc : dml.data.conditions) {
         Condition.LHS lhs = createLHS(bc.lhs());
-        TypedValue v = handleValue(lhs.toString(), lhs.valueType(), bc.value());
+        TypedValue v;
+        if (bc.predicate().equals(Predicate.IN)) {
+          // IN works only on a LIST type
+          v = handleValue(lhs.toString(), Column.Type.List.of(lhs.valueType()), bc.value());
+        } else if (bc.predicate().equals(Predicate.CONTAINS)) {
```

Comment:
CONTAINS and CONTAINS `bind` was also broken and not tested. I've fixed both of those